### DISCUSSION TITLE
zotero插件

### DISCUSSION THREAD

**Discussion by anshangtangyuan** [immersive-translate/immersive-translate#1876] [Help]
### 概述 | Summary

可不可以增加zotero插件

**Comment by liruifengv** [comment]
您好，我们的 Zotero 插件发布了，目前 0.0.1 版本，欢迎下载使用一下，在 [Release](https://github.com/immersive-translate/zotero-immersivetranslate/releases) 下载 xpi 安装

https://github.com/immersive-translate/zotero-immersivetranslate

**Reply by anshangtangyuan** [reply]
OK
